recipes-core/acrn/files/paths.patch

```diff
@@ -1,6 +1,6 @@
-From 63a9da3bb8369abe50572ff1abd812f4fb3d9300 Mon Sep 17 00:00:00 2001
-From: Ross Burton <ross.burton@intel.com>
-Date: Tue, 15 Jan 2019 14:49:48 +0000
+From 05e8a1013695bbdb25889b6adcfa0eb59b6ac5ba Mon Sep 17 00:00:00 2001
```

Comment:
*Really* need to get at least some of this upstream as rebasing it every release is getting very boring.

Reply:
Has that ever been submitted upstream yet? (I don't recall seeing it but I may have missed it too)

Reply:
I filed https://github.com/projectacrn/acrn-hypervisor/issues/2347 and was told in May it is being worked on.  I think we should just submit this patch as-in, it's no perfect but better than nothing.

Reply:
> I filed [projectacrn/acrn-hypervisor#2347](https://github.com/projectacrn/acrn-hypervisor/issues/2347) and was told in May it is being worked on. I think we should just submit this patch as-in, it's no perfect but better than nothing.

I agree 👍